fabric-item-groups-v0/src/main/resources/assets/fabric/lang/lzh.json

```diff
@@ -0,0 +1,3 @@
+{
+  "fabric.gui.creativeTabPage": "卷%2$s之%1$s"
```

Comment:
> I'm just following the vanilla convention. In vanilla assets files, `zh-cn.json` usually does not leave spaces between character and words, while `zh-hk.json` and `zh_tw.json` do.

This one has inconsistency with the other `lzh.json` then

Reply:
That's also the convention of vanilla's `lzh.json`, where spaces are usually not left. And actually whether to leave space does not matter—both behaviours are acceptable.

Reply:
So you add space for english but not number? sure... cursed enough